src/transform/shot_count_block/qir.rs

```diff
@@ -405,5 +421,6 @@ mod test {
         make_snapshot_test!(reduction);
         make_snapshot_test!(vqe_iteration);
         make_snapshot_test!(cartesian_rotations);
+        make_snapshot_test!(transformed_with_qcs_decls);
```

Comment:
```suggestion
        make_snapshot_test!(constant_parameter_values);
```

Looks like this was a holdover from an earlier change, but as you add it and its snapshots please update the name to something expressive

Reply:
👌 neither are in tree after rebase -- good eye